src/service/run-api-service.js

```diff
@@ -12,6 +12,7 @@
  *
  * * `account`: Epicenter account id (**Team ID** for team projects, **User ID** for personal projects).
  * * `project`: Epicenter project id.
+ * * `id`: (Optional) The run id of an existing run. If there is a specific run you'd like to work with, you can include it.
```

Comment:
rs.query doesn't respect id though (either passed to the call or passed to the instantiation of Run Service). we can look at examples when we meet shortly?